operator/cmd/mesh/manifest-apply.go

```diff
@@ -119,10 +132,37 @@ func ApplyManifests(setOverlay []string, inFilenames []string, force bool, dryRu
 	if err != nil {
 		return err
 	}
+	client, err := kubernetes.NewForConfig(kubeconfig)
```

Comment:
I'm not familiar with operator - but:
- isn't a client created already somewhere ( since kubeconfig is passed ) ? We don't want to create too many clients, generally reusing one is better AFAIK.
- in dryRun - shouldn't it not connect to k8s ?

Reply:
fixed